### ISSUE TITLE
[Feature] Clicking on notification opens the chat

### ISSUE THREAD

**Issue by maicol07** [amanharwara/altus#87]
Hi,

currently if I click on the notification it only disappears. It would be great if clicking on that opens Altus in that chat (and in that message).

Thanks

P.S.: There is also a minor issue where the notification has electron.Altus instead of Altus and no icon

**Comment by servitecbaires** [comment]
Hello! Im using Xubuntu 20.04 and the appImage release. I have the same problem.
The new message notification only diappears on click.

Thanks

**Comment by amanharwara** [comment]
Implemented in [v3.18.0](https://github.com/amanharwara/altus/releases/tag/3.18.0)

**Comment by maicol07** [comment]
It seems that the notification opens Altus but on the last chat, not on the message of the notification

**Comment by amanharwara** [comment]
Not sure if I'll be able to implement that. It worked previously because I wasn't using my own click event, the notification was using WhatsApp's default click event. Now that I'm using my own click event to make the window maximized on click, that functionality doesn't work. I might be able to implement it but it won't be the same what WhatsApp did before.

**Comment by amanharwara** [comment]
Implemented in [v3.19.0](https://github.com/amanharwara/altus/releases/tag/3.19.0)

**Comment by maicol07** [comment]
@amanharwara it doesn't seem to work...

**Comment by amanharwara** [comment]
Hmm that's weird. I tested in on two windows 10 machines and a macOS instance and it worked fine every time. Do you have any idea as to what could be causing it to not work?

**Comment by maicol07** [comment]
Unfortunately not at the moment... I'll update this if I'll find some

**Comment by amanharwara** [comment]
Any update on this? Are the notifications working correctly?

**Comment by maicol07** [comment]
Yes, I think they works better. I only have some notes about the notifications:

*   When app is minimized in tray or apps bar notifications should be persistent (at the moment when a notifications expires its _foreground time_ it's not sent to notifications center)
*   When Altus is open all persistent notifications should be cleared and the new notifications (while the app is opened and in foreground) notifications should be temporary, like they are now

**Comment by cobrabr** [comment]
I'm on Windows 10, and clicking the notification doesn't open Altus at all.

**Comment by cobrabr** [comment]
Just an update: I'm on Windows 11 now, and clicking the notification still doesn't open Altus.

**Comment by chaoscreater** [comment]
> Just an update: I'm on Windows 11 now, and clicking the notification still doesn't open Altus.

Same issue on latest version v5.1 on Windows 11

**Comment by CloudedQuartz** [comment]
> Just an update: I'm on Windows 11 now, and clicking the notification still doesn't open Altus.

Just discovered Altus, same issue on win 11 v5.8
